components/Navbar/index.tsx

```diff
@@ -151,6 +154,13 @@ function Navbar() {
             {isConnected && chain?.id === goerli.id && (
               <Chip label="Goerli Faucet" onClick={handleFaucetClick} sx={{ my: 'auto', display: onlyDesktopFlex }} />
             )}
+            {isOPMainnet && (
+              <Link href={'/bridge'}>
+                <Button variant="contained" sx={{ minWidth: '125px' }}>
+                  {t('Bridge & Swap')}
+                </Button>
+              </Link>
+            )}
```

Comment:
This is a minor thing. When on `/bridge` the Markets button is still contained as we were on that page which is incorrect